### PR TITLE
JAMES-2511 Maven issues many warning before compilation

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3202,185 +3177,6 @@
...
-                                                the report as part of a distribution, which is licensed under the ASL, or a similar license,
-                                                which is incompatible with the GPL
-                                            -->
-                                            <omitGplFiles>true</omitGplFiles>
-                                        </configuration>
-                                    </plugin>
-                                    <plugin>
-                                        <groupId>org.apache.rat</groupId>
-                                        <artifactId>apache-rat-plugin</artifactId>
-                                        <configuration>
```
Comment (mbaechler) [inline pom.xml]: why did you removed most configuration ?

File: mpt/mavenplugin/pom.xml [hunk 2]
```diff
@@ -208,4 +200,13 @@ to the library requiring no extra coding.</description>
             </build>
         </profile>
     </profiles>
+
+    <reporting>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-plugin-plugin</artifactId>
```
Comment (mbaechler) [inline mpt/mavenplugin/pom.xml]: are you sure it does the combine.children="append" ?

File: pom.xml (hunk 1 above)
Comment (chibenwa) [reply pom.xml]: Configuration is not allowed in the reporting section.

File: mpt/mavenplugin/pom.xml (hunk 2 above)
Comment (chibenwa) [reply mpt/mavenplugin/pom.xml]: I have no idea of how to merge reports plugin together.

https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Configuring_Reporting_Plugins seems to suggest inheritance is the default. I would not worry about this.

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**